src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -28,6 +28,25 @@ typedef NS_ENUM(NSInteger, MTRCommissioningStatus) {
     = 3,
 } API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * Read from controllee's Information  at Commissioning
```

Comment:
That's not part of the interface definition....  How about:
```suggestion
 * A representation of a (vendor, product) pair that identifies a specific product.
```